File: CS/Helpers/DiagramLayoutWrapper.cs
using System.Collections.Generic;
using System.Xml.Serialization;

namespace MDI_Diagram {
    [XmlRoot("Root")]
    public class DiagramLayoutWrapper {
        [XmlArray("Items")]
        [XmlArrayItem("Item")]
        public List<string> Diagrams { get; set; } = new List<string>();
    }
}

File: VB/Helpers/DiagramLayoutWrapper.vb
Imports System.Collections.Generic
Imports System.Xml.Serialization

Namespace MDI_Diagram

    <XmlRoot("Root")>
    Public Class DiagramLayoutWrapper

        <XmlArray("Items")>
        <XmlArrayItem("Item")>
        Public Property Diagrams As List(Of String) = New List(Of String)()
    End Class
End Namespace
